crates/types/src/rpc/transactions.rs

```diff
@@ -435,11 +435,11 @@ pub struct SimulatedTransaction {
 
 impl FunctionInvocation {
     pub fn try_from_call_info(
-        mut call_info: blockifier::execution::call_info::CallInfo,
+        call_info: &blockifier::execution::call_info::CallInfo,
```

Comment:
Why was this needed?

Reply:
`starknet_traceTransaction` is invoked for already executed transactions. They are stored in collection and getting them from it returns a reference. The reason for changing the signature of the method is that `blockifier::execution::call_info::CallInfo` doesnt implement Copy/Clone and using it with an object that is a reference struct results in `moving out of shared reference` error.